parser/model/model.go

```diff
@@ -2152,3 +2152,11 @@ func (s WindowRepeatType) String() string {
 		return ""
 	}
 }
+
+// TraceInfo is the information for trace
```

Comment:
```suggestion
// TraceInfo is the information for trace.
```
Ditto for the rest of comments.